salt/fileserver/s3fs.py

```diff
@@ -57,6 +57,25 @@
 
 .. note:: This fileserver back-end requires the use of the MD5 hashing algorithm.
     MD5 may not be compliant with all security policies.
+
+.. note:: This fileserver back-end is only compatible with MD5 ETag hashes in
+    the S3 metadata. This means that you must use SSE-S3 or plaintext for
+    bucket encryption, and that you must not use multipart upload when
+    uploading to your bucket. More information here:
+    https://docs.aws.amazon.com/AmazonS3/latest/API/RESTCommonResponseHeaders.html
+
+    Objects without an MD5 ETag will be fetched on every fileserver update.
+
+    If you deal with objects greater than 8MB, then you should use the
+    following AWS CLI config to avoid mutipart upload:
+
+    .. code-block::
+
+        s3 =
+          multipart_threshold = 1024MB
+
```

Comment:
Was the line break and indentation after the `=` intentional? Just making sure as I am not familiar with this syntax.

Reply:
I think so. You can see example config here: https://docs.aws.amazon.com/cli/latest/topic/s3-config.html

I just realized that I accidentally used the same link twice -- the second instance should be that link above. I'll fix that momentarily.